lib/Agrammon/Web/Routes.pm6

```diff
@@ -105,7 +105,24 @@ sub api-routes (Str $schema, $ws) {
                 }
             }
         }
-        # working
+        operation 'cloneDataset', -> LoggedIn $user {
+            request-body -> ( :oldUsername($old-username), :newUsername($new-username),
```

Comment:
Should one really be able to specify the username of the user that this is cloned to from the outside? Is creating a dataset on any another user's account a desired feature, or could it be a security issue?

Reply:
This is used for giving a copy of one‘s own dataset(s) to another user.

Reply:
In the other hand, I‘ll have to check that one cannily give one‘s own datasets away.